ts/nni_manager/training_service/common/util.ts

```diff
@@ -95,6 +95,16 @@ export async function execMkdir(directory: string, share: boolean = false): Prom
     return Promise.resolve();
 }
 
+export async function fileExist(filePath: string): Promise<boolean> {
```

Comment:
What's the problem of `fs.access()`?

Reply:
refactored code, removed fileExist function